app/models/user.rb

```diff
@@ -28,4 +28,12 @@ def avatar_url(size)
     gravatar_id = Digest::MD5::hexdigest(self.email).downcase
     "http://gravatar.com/avatar/#{gravatar_id}.png?s=#{size}"
   end
+
+  def has_comments?
+    comments.count > 0
+  end
+
+  def has_posts?
+    posts.count > 0
+  end
```

Comment:
Awesome. Great choice to use `user.rb` as the place for these methods. Also, yep, this is how I would have written them.